wdqs-frontend/latest/.travis/build-deploy.sh

```diff
@@ -2,3 +2,7 @@
 #Oneline for full directory name see: https://stackoverflow.com/questions/59895/getting-the-source-directory-of-a-bash-script-from-within
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 docker build "$DIR/../" -t wikibase/wdqs-frontend:latest
+
+if [ "$BRANCH" == "master" ]
```

Comment:
Rather than use $BRANCH everywhere it might be nice to set a new env var called "SHOULD_DOCKER_PUSH" or something similar?

Reply:
Yes, good idea. Will do